codecs/src/aper/decode/decode_charstrings.rs

```diff
@@ -72,49 +35,7 @@ pub fn decode_printable_string(
         ub,
         is_extensible
     );
-
-    let (_val_lower, _val_higher) = (32u8, 122u8);
-
-    let num_bits = 8; // N = 74, B = 7, B2 = 8
-
-    let mut alphabet = vec![' ', '\'', '(', ')', '+', ',', '-', '.', '/'];
```

Comment:
I am surprised this alphabet is not used later and I didn't see any warnings too! :-(